backend/scripts/load_data.py

```diff
@@ -55,12 +24,75 @@ def load_sample_data():
     Returns:
       None
     """
-    for gpkg in glob(f"{DATA_DIR}/*.gpkg"):
-        subprocess.run(["bash", "./scripts/load_gerrydb_view.sh", gpkg])
+
+    subprocess.run(["alembic", "upgrade", "head"], cwd=path.join(currentdir, ".."))
+    for view in config["gerrydb_views"]:
+        subprocess.run(
```

Comment:
PP: for each of these subprocess runs, you can use the util functions rather than CLI calls. The CLI is a simple wrapper around those util functions

Reply:
Yeah totally fine for now. That was a "PP" (personal preference) non-blocking comment. And yeah the click functions can't be called but they're simple wrappers around functions in `app/utils`, which _can_ be called directly

Reply:
Which is actually preferable because since you have to pass them a session, you can run them all in a single transaction, just `session.commit()` at the end. Anyway at some point (in diff PR) I can go in and make those changes